helm/alfresco-content-services/charts/alfresco-search/README.md

```diff
@@ -37,7 +38,7 @@ A Helm chart for deploying Alfresco Search
 | insightEngineImage.internalPort | int | `8983` |  |
 | insightEngineImage.pullPolicy | string | `"IfNotPresent"` |  |
 | insightEngineImage.repository | string | `"quay.io/alfresco/insight-engine"` |  |
-| insightEngineImage.tag | string | `"2.0.0"` |  |
+| insightEngineImage.tag | string | `"2.1.0-SNAPSHOT"` |  |
```

Comment:
You are probably correct, but I thought it was 2.0.3 rather than 2.1.0 https://alfresco.atlassian.net/browse/ACS-2305 - Going to the next minor makes sense for this change.

Reply:
The changes in this branch needed the Docker image support for `ALFRESCO_SECURE_COMMS=secret`, which was originally only available on the latest `2.1.0-SNAPSHOT` build. Since the [IE/SS release build](https://app.travis-ci.com/github/Alfresco/InsightEngine/jobs/561786320) finally ran yesterday, I think it'd be safe to test with `2.0.3-RC2`, which should include the aforementioned Docker image support.